### PR TITLE
chore(deps): use web_bson for bson library

### PR DESCRIPTION
**Pull request by erfanium**
[web_bson](https://deno.land/x/web_bson@v0.1.0) is a fork of [js-bson](https://github.com/mongodb/js-bson) for Deno and has been made on top of web apis, so it's faster and lighter than js-bson.

### CODE REVIEW

**Review by lucsoft — APPROVED** [review]
LGTM :shipit:

File: deps.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
-export * as Bson from "./bson/bson.ts";
-export { crypto } from "https://deno.land/std@0.117.0/crypto/mod.ts";
-export { BufReader } from "https://deno.land/std@0.117.0/io/mod.ts";
-export { writeAll } from "https://deno.land/std@0.117.0/streams/conversion.ts";
-export { deferred } from "https://deno.land/std@0.117.0/async/deferred.ts";
-export type { Deferred } from "https://deno.land/std@0.117.0/async/deferred.ts";
-export * as b64 from "https://deno.land/std@0.117.0/encoding/base64.ts";
-export * as hex from "https://deno.land/std@0.117.0/encoding/hex.ts";
+export * as Bson from "https://deno.land/x/web_bson@v0.1.2/mod.ts";
```
Comment (lucsoft) [inline deps.ts:1]:
```suggestion
export * from "https://deno.land/x/web_bson@v0.1.2/mod.ts";
```
the MongoDB driver also doesn't wrap it. We should also not wrap it.

**Review by lucsoft — CHANGES_REQUESTED** [review]

File: deps.ts (hunk 1 above)
Comment (erfanium) [reply deps.ts:1]: We can't decide here. i think warping Bson types makes the driver more clean. so better to move this discussion into a separated issue.

**Comment by erfanium** [comment]
@manyuanrong 
I will make a beta version

**Review by lucsoft — APPROVED** [review]
:shipit:

**Pull request merged**